packages/studio-web/src/app/demo/demo.component.sass

```diff
@@ -3,3 +3,4 @@
     button.download
         margin-top: 5px
         margin-left: 15px
+    justify-content: flex-end
```

Comment:
Oy vey! IU tried with flex and padding and align and margin. Very CSS. There's one trivial thing that's needed, but good luck figuring out which one!